project/EnsimeBuild.scala

```diff
@@ -286,6 +287,10 @@ object EnsimeTestingBuild {
     libraryDependencies += "org.scalatest" %% "scalatest" % "3.0.1" % Test intransitive ()
   )
 
+  lazy val testingSimple2 = testingProject("testing/simple2") settings (
```

Comment:
ok, but you can do that already by filtering the `.ensime` to include two named projects.